### DISCUSSION TITLE
pouch inside spellbook

### DISCUSSION THREAD

**Discussion by ShadoweCZ** [Truinto/SpellPouch#10] [Ideas]
im wondering if it would be possible to creat the pouches inside the spellbook tray (the same way it can be done inside Abilities tray)

im asking for this especialy because of tier 0 spells that if you combine more classes can become a mess, now add to that special class abilities like alchemist being able to share personal buffs or arcanist having supperior buffs on transmutation spells ... basicly would be easier to bundle each class spells separately ort even sort them by class and school etc ... 

also when im writing here ther eseems to be small bug with some predefined pouches ... basicly abilities that are pouches themselves (Arcane reservoir, Tactician etc) skills  that  you have to choose "version off" dont work when they are in pouch ... basicly  u cannot open pouch (and choose the skill version) inside pouch

**Comment by Truinto** [comment]
Anything is possible. It's rather a question of complexity and time.

I made abilities group globally, because the main goal was to reduce clutter and the list overflowing with too many entries.
Grouping spells was an idea that came up upon first release by someone on Discord. It is a great idea, so I did that as well. But spells are a lot more complex, since you can have many spellbooks with the same spell and metamagic spells. So it made sense to just have the pouch in the hotbar. It's also nice, because the hotbar automatically handles the data for saving. I can't save data in the spellbook. Obviously that can be worked around, somehow. But I see no purpose, if you had to do that for each character. And if it's global, that's ugly too since you probably want different load-outs for different characters. So in my mind it just makes little sense.

About the bug, you probably mean that spells can't be unfolded when they are in the pouch. The base UI is written in a way that only one icon can be unfolded at the same time. I am sure there is a work around I haven't found yet. Tho I am happy it works as well as it does now :)
